### ISSUE TITLE
Error downloading 7 generations

### ISSUE THREAD

**Issue by clampak** [Linekio/getmyancestors#5]
Or does it allow 7 generations?  Is there a limit?  What happens when it reaches a generation without ancestors?  I have >7 generations in my tree, and was just trying this out.   It's the current version of the script, which I just found out about and installed today.  Below is the output.

```
python3 getmyancestors.py -a 7 -d 2 -m -o out7.ged -l out7.log -v
Enter FamilySearch username: 
Enter FamilySearch password:
Login to FamilySearch...
Download starting individuals...
Download 1th generation of ancestors...
Download 2th generation of ancestors...
Download 3th generation of ancestors...
Download 4th generation of ancestors...
Download 5th generation of ancestors...
Traceback (most recent call last):
  File "getmyancestors.py", line 943, in <module>
    todo = tree.add_parents(todo) - done
  File "getmyancestors.py", line 790, in add_parents
    self.add_indis(parents)
  File "getmyancestors.py", line 743, in add_indis
    loop.run_until_complete(add_datas(loop, data))
  File "/usr/lib/python3.6/asyncio/base_events.py", line 467, in run_until_complete
    return future.result()
  File "getmyancestors.py", line 730, in add_datas
    await future
  File "/usr/lib/python3.6/concurrent/futures/thread.py", line 56, in run
    result = self.fn(*self.args, **self.kwargs)
  File "getmyancestors.py", line 511, in add_data
    self.memories.add(Memorie(x))
  File "getmyancestors.py", line 364, in __init__
    self.url = data['links']['alternate'][0]['href']
KeyError: 'alternate'
Future exception was never retrieved
future: <Future finished exception=KeyError('alternate',)>
Traceback (most recent call last):
  File "/usr/lib/python3.6/concurrent/futures/thread.py", line 56, in run
    result = self.fn(*self.args, **self.kwargs)
  File "getmyancestors.py", line 511, in add_data
    self.memories.add(Memorie(x))
  File "getmyancestors.py", line 364, in __init__
    self.url = data['links']['alternate'][0]['href']
KeyError: 'alternate'

**Comment by Linekio** [comment]
There is no limit. Please send me the log file.
Cordialement,
Benoît Fontaine
Tél : +33 6 32 71 10 08

**Comment by clampak** [comment]
Sent via email.

**Comment by Linekio** [comment]
The error is due to Familysearch who made changes to the memorie's
features. I just added a temporary fix, please download and retry :
https://github.com/Linekio/getmyancestors
I'll add the changes later to handle all types of memories.

Cordialement,
Benoît Fontaine
Tél : +33 6 32 71 10 08

**Comment by clampak** [comment]
Thank you, this did work for me.

**Comment by Linekio** [comment]
Fixed with ac3135adc2ec5d5e93fc3c7541af69ac422ba5f5 and 828c8d096bfaff56f238fc1607e0671ab231c78b
